modules/custom_role_iam/README.md

```diff
@@ -0,0 +1,62 @@
+# Module Custom Role IAM
+
+This optional module is used to create custom roles at organization or project level.
+
+## Usage - Custom Role at Organization Level
+
+```hcl
```

Comment:
Please update examples to use right inputs/outputs and not use locals.

Reply:
Fixed it.